sh/travis.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# change to the top-level bhima directory
+#cd ..
+
+# build the application
+gulp build
+
+# lint the client
+gulp lint
+
+# lint the server
+jshint server/{controllers,lib,config}/*.js
```

Comment:
Yes, I agree.  Considering that the server (potentially) going to change a fair amount, this is a quick-n-dirty way to get the job done.